GooglePlayInstant/Editor/PlayInstantIl2cppBuilder.cs

```diff
@@ -0,0 +1,75 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// A handler for building apks using quick deploy.
+    /// </summary>
+    public static class Il2cppBuilder
+    {
+        private const string SplashScenePath = "play-instant-loading-screen-scene.unity";
```

Comment:
Is this constant already declared somewhere else in the code base? If so please make it public and reuse it.